functions/Clear-DbaSqlConnectionPool.ps1

```diff
@@ -44,31 +44,31 @@ function Clear-DbaSqlConnectionPool {
 		[Parameter(ValueFromPipeline = $true)]
 		[Alias("cn", "host", "Server")]
 		[DbaInstanceParameter[]]$ComputerName = $env:COMPUTERNAME,
-		[PSCredential]$Credential,
+		[PSCredential][System.Management.Automation.Credential()]$Credential = [System.Management.Automation.PSCredential]::Empty,
 		[switch]$Silent
 	)
 
 	process
 	{
 		# TODO: https://jamessdixon.wordpress.com/2013/01/22/ado-net-and-connection-pooling
 
-		foreach ($Computer in $ComputerName)
+		foreach ($computer in $ComputerName)
```

Comment:
This one can stay as lower case since it is an internal variable. The only ones we want to capitalize the first letter will be the parameters for the function.